shell.nix

```diff
@@ -82,6 +82,7 @@ llvmPackages_14.stdenv.mkDerivation {
 
   CARGO_TERM_COLOR = "always";
   RUST_BACKTRACE = "full";
+  RUSTFLAGS = "--cfg tokio_unstable";
```

Comment:
oh awesome, yes!

i'd done this via `.cargo/config`, but this is easier :slightly_smiling_face: 

related:

* https://github.com/linkerd/linkerd2-proxy/pull/3443
* https://github.com/linkerd/linkerd2-proxy/pull/3451